docker/dev/dev_entrypoint.sh

```diff
@@ -13,10 +13,10 @@ fi
 
 echo "Collecting static..."
 
-while ! python manage.py collectstatic --no-input 2>&1; do
-  echo "Collecting static..."
-  sleep 3
-done
+#while ! python manage.py collectstatic --no-input 2>&1; do
```

Comment:
collectstatic, zależnie od serwera na którym uruchamiany jest projekt, może kopiować statyczne pliki tak żeby nie były serwowane przez pythona tylko bezpośrednio. (wźcej poszukaj info o django collectstatic) 

Nie wiem jak chłopaki to u siebie organizują w dev, ale też nie wywalałbym im tego bez porozumienia - jak Ci to sprawiało problemy to ich zapytaj.

ewentualnie cały katalog `docker/dev` można skopiować na `docker/oca_dev`  i tam hulaj dusza, ale i tak trzeba zromieć zarządanie staticami przez django generalnie (z doc) i w kontekście bhtoma (od chłopaków)

Reply:
Nie kopiujemy static, używamy ścieżki w której one są, udostępniamy je przez nginx location i alias. Co do drugiej sprawy, może faktycznie lepiej by było docker/oca_dev, bo trudno powiedzieć, w którym kierunku będą szły nasze zmiany w przyszłości - ale nie upieram się.